### ISSUE TITLE
add stackage-server to Hackage?

### ISSUE THREAD

**Issue by juhp** [fpco/stackage-server#221]
How about publishing stackage-server to Hackage, then it could be included in Stackage too?

**Comment by snoyberg** [comment]
I'm not opposed, but the Hackage trustee team has put a lot of pressure on me to not release packages to Hackage that won't include bounds, so I'd rather not pick a fight for no reason.

**Comment by alanz** [comment]
A quick look at the cabal file shows upper bounds on everything? Or is it the dependencies you are worried about?

**Comment by snoyberg** [comment]
Huh, imagine that. It's because it's inherited from the Yesod templates, which include bounds. However:

1. The lower bounds are certainly not maintained correctly
2. I've never intended on keeping the upper bounds, and may remove them in the future

An application like stackage-server is the prime example IMO where bounds serve no purpose, since the only approved way of building is via the stack.yaml config.

All this said: I don't mind if someone wants to upload to Hackage and maintain it there, but I'll put in 0 effort on fixing Hackage matrix failures, as I could not care less about mass compatibility for an application.

**Comment by alanz** [comment]
Fair enough. For interest, with cabal 1.24 it does not find a build plan currently, the lens upper bound is too restrictive.
cabal master does find it though, so that is encouraging for the future.

If anyone cares enough to upload it to stackage, I am prepared to be listed as the guy to do PRs to maintain the upper bounds.

But if it does not go onto hackage it does not bug me.
